shared/src/persy/kv_index.rs

```diff
@@ -1,16 +1,17 @@
 use persy::{ByteVec, IndexType, PRes, ValueMode};
 use serde::{de::DeserializeOwned, Serialize};
 
-use super::{Index, PersyConnection};
+use super::{Connection, Index};
 
-pub struct KeyValueIndex<'a, K> {
+pub struct KvIndex<'a, K> {
```

Comment:
```suggestion
pub struct KVIndex<'a, K> {
```

Since both letters are the starts of words, they should both be capitalized.

Reply:
Sigh I'm wrong. https://doc.rust-lang.org/1.0.0/style/style/naming/README.html

Reply:
This documentation is deprecated.
Here is the new one:
https://rust-lang.github.io/api-guidelines/naming.html#casing-conforms-to-rfc-430-c-case

I actually found out about this rule just a couple of weeks ago.